target/linux/lantiq/image/Makefile

```diff
@@ -291,9 +323,12 @@ endef
 TARGET_DEVICES += arcadyan_arv7506pw11
 
 define Device/arcadyan_arv7510pw22
+  DEVICE_VENDOR := Arcadyan
+  DEVICE_MODEL := ARV7510PW22
+  DEVICE_ALT0_VENDOR := Astoria networks
+  DEVICE_ALT0_MODEL := ARV7510PW22
```

Comment:
For those, I would have actually just used brackets or slash:
`DEVICE_VENDOR := Arcadyan (Astoria networks)`
But your solution is equally fine.

Reply:
I would stay with the current state. It fits the scheme for the other models.